karapace/schema_registry_apis.py

```diff
@@ -55,6 +60,7 @@ class SchemaErrorMessages(Enum):
         "forward, full, backward_transitive, forward_transitive, and "
         "full_transitive"
     )
+    REFERENCES_SUPPORT_NOT_IMPLEMENTED = "Schema references are not supported for '{schema_type}' schema type yet"
```

Comment:
The `yet` is not needed and the request could contain arbitrary string as schema type resulting `Schema references are not supported for '<ARBITRARY>' schema type yet`

Reply:
The request always contains JSON with the exact schema type definition from the set of AVRO, JSON, and PROTOBUF in other case we raise an error.

Reply:
I meant that `yet` points to future support. I would just state in the error that not supported without hinting on possible future support.